rsmtool/test_utils.py

```diff
@@ -157,10 +169,10 @@ def check_csv_output(csv1, csv2):
     # convert any integer columns to floats in either data frame
     for df in [df1, df2]:
         for c in df.columns:
-            if df[c].dtype == np.int64:
+            if not c in string_columns and df[c].dtype == np.int64:
```

Comment:
if you are reading them as strings to begin with, then do we need this test at all?